docs/dev/release-procedure.rst

```diff
@@ -64,6 +64,18 @@ First, test all the steps first with the `unstable-repository <http://build.open
 * Build locally for all distributions.
 * Commit.
 
+************
+Docker Image
+************
+
+Releasing a new Docker image is very easy.
+
+* Clone `IntelMQ Docker Repository <https://github.com/certat/intelmq-docker>`_
+* Set `chmod +x build.sh`, `chmod +x test.sh` & `chmod +x publish.sh`
+* Run `sudo ./build.sh`, check your console if the build was successful.
+* Run `sudo ./test.sh` - It will run nosetests3 with exotic flag. All errors/warnings will be displayed.
+* If no error/warning was shown, you can release with `sudo ./publish.sh` **ATTENTION** Only available for CERT.AT employee's.
```

Comment:
I'm pretty sure that the publishing of a container images does not require superuser privileges